content/guides/models/automations/_index.md

```diff
@@ -3,7 +3,341 @@ menu:
+
+## Event scopes and types
+
+An automation's triggering event depends on the automation's scope.
+
+### Registry
+For a [registered model]({{< relref "/guides/models/registry/">}}), you can configure an automation to run on these events:
+
+- **Linking a new artifact to a registered model**: Test and validate new model candidates.
+- **Adding a new alias to a version of the registered model**: Trigger a special step of your workflow when a new model version has a label or alias applied. For example, deploy a model when it has the `deploy` alias applied.
+
+### Project
+For a model in a project that is not in a registry, you can configure an automation to run on these events:
+
+- **A new version of an artifact is created in a collection**: Apply recurring actions to each version of an artifact. For example, start a training job when a new dataset artifact version is created.
+- **An artifact alias is added**: Apply recurring actions when a specific alias is applied to an artifact version. For example, run a series of downstream processing steps when an artifact gains the `test-set-quality-check` alias.
+
+### Run
+You can configure an automation to run based on the results of a run:
+
+- **Run metric threshold met**: Apply recurring actions based on a run metric's value and whether it is above, below, or exactly equal to the threshold. Optionally, specify how to average results.
+- **Run metric change threshold met**: Apply recurring actions based on whether a run metric's amount of change (absolute or relative) is above, below, or exactly equal to the threshold. Optionally, specify how to average results.
+
+## Evant actions
+An automation can run a webhook on a third-party service such as GitHub or Microsoft Teams, or it can post to a Slack channel.
+
+## Create an automation
+These sections show how to configure a Slack automation or a webhook automation.
+
+### Configure a Slack automation
+Configuring a Slack integration takes multiple steps. First, [add one or more Slack channels]({{< relref "#add-a-slack-channel" >}}) as alert destinations. Next, [create the automation that notifies the Slack channel]({{< relref "#create-slack-automation" >}}).
+
+#### Add a Slack channel
+
+1. Go to Team Settings page.
+1. To integrate with a new Slack workspace and channel, click **Connect Slack**.
+    {{% alert %}}
+    To integrate with a new Slack channel in a workspace that is already set up, click **New integration**.
+    {{% /alert %}}
+
+  If necessary, sign in to your Slack instance. A browser window appears, asking that you grant Weights and Biases permission to post to the Slack channel you select. Read the page, then click **Search for a channel** and begin typing the channel name. Select the channel from the list, then click **Allow**.
+
+1. In Slack, go to the channel you selected. If you see a post like `[Your Slack handle] added an integration to this channel: Weights & Biases`, the integration is configured correctly.
```

Comment:
@tonyyli-wandb is there any action that admins need to take to approve the integration?

Reply:
In my testing, I did not need anyone to approve it and I am not a Slack admin. I think Slack just posts using the permissions of the Slack user enabling the integration.